src/SFML/Graphics/Texture.cpp

```diff
@@ -441,6 +454,94 @@ void Texture::update(const Uint8* pixels, unsigned int width, unsigned int heigh
 
 
 ////////////////////////////////////////////////////////////
+void Texture::update(const Texture& texture)
+{
+    // Update the whole texture
+    update(texture, 0, 0);
+}
+
+
+////////////////////////////////////////////////////////////
+void Texture::update(const Texture& texture, unsigned int x, unsigned int y)
```

Comment:
I think for now the way @binary1248 did it is good, but we should reflect on this for 3.0.